docs/_static/vcr/v2.yaml

```diff
@@ -10,6 +10,72 @@ info:
 servers:
   - url: http://localhost:1323
 paths:
+  /internal/vcr/v2/holder/vc/search:
```

Comment:
How it matching performed? Do you need to pass the exact VC (kinda defeats the purpose of the search function)?

Reply:
> Each field in the request body must be present in the VC in order for it to be passed as result.
So the less you enter in the request body, the more it finds.